README.md

```diff
@@ -15,6 +15,12 @@ pip install aioswitcher
   <td><a href="https://github.com/TomerFi/aioswitcher/blob/dev/CONTRIBUTING.md">Contributing</a></td>
 </table>
 
+<strong>
+NOTE: For newer Switcher devices such as: Runner S11, Runner S12 and Lights you need to have a Token for communicating with devices.
```

Comment:
Which model are we adding support to? PR description is S11, comment here mention S12 and lights

Reply:
@thecode it's just a comment for the future which is true for all the devices mentioned there.
I can leave only Runner S11 but I will need to add those anyway when we continue to support S12 and Lights..

Reply:
I prefer that the docs will reflect the current situation, someone might read it and open an issue why it doesn't work for S12

Reply:
Done. I will commit the updated code soon.

Reply:
Done. You can resolve it.